README.md

```diff
@@ -266,13 +304,13 @@ Below are the commands you will most commonly use, note that these have to be ru
 
 ```bash
 # Check if you need to generate any new migrations after changes to the model
-poetry run django-admin makemigrations --check
+make check-makemigrations
 
 # Generate migrations
-poetry run django-admin makemigrations
+make makemigrations
 
 # Apply migrations. Needed if new migrations have been generated (either by you, or via upstream code)
-poetry run django-admin migrate
+make migrate
```

Comment:
I suggest replacing all the `dj` alias commands like `djrun` to use the makefile target instead so we have a consistent commands to run regardless of whether we are running via docker or locally.

Reply:
I really don't have a strong preference haha.

I mean I agree with the approach that @MebinAbraham is suggesting but when it comes to removing them from `/.docker/bashrc.sh` it makes to get rid of it, unless there is a solid reason to keep those still.

Reply:
The `dj` aliases are something we've been using at Torchbox as a shortcut, so not wedded to it